crates/dbs-virtio-devices/src/block/device.rs

```diff
@@ -221,11 +221,11 @@ where
     }
 
     fn read_config(&mut self, offset: u64, data: &mut [u8]) {
```

Comment:
should we change VirtioDevice trait to return bool in read_config()?

Reply:
yes. I think so, although it hasn't been used yet.

Reply:
You can have a try

Reply:
I tested it in the test.